tests/drivers/gpio/gpio_basic_api/boards/frdm_ke15z.overlay

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright NXP 2024
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&gpiod {
+	status = "disabled";
+};
+
+&gpioc {
+	interrupts = <26 2>;
+	status = "okay";
+};
+
+/ {
+	resources {
+		compatible = "test-gpio-basic-api";
+		out-gpios = <&gpioc 8 0>; /* Arduino A0 */
+		in-gpios  = <&gpioc 9 0>; /* Arduino A1 */
```

Comment:
optional but could add an arduino connector nexus node in DT to make it simpler for users